doc/lua_api.md

```diff
@@ -9048,6 +9050,23 @@ child will follow movement and rotation of that bone.
             * Currently, bloom `intensity` and `strength_factor` affect volumetric
               lighting `strength` and vice versa. This behavior is to be changed
               in the future, do not rely on it.
+      * `color_transform_matrix`: is a matrix with default value (identity matrix):
+        ```lua
+          { {1.0, 0.0, 0.0}, -- r
+            {0.0, 1.0, 0.0}, -- g
+            {0.0, 0.0, 1.0}} -- b
+        ```
+
+        * Work as `transformed_color_RGB = color_transform_matrix * color_RGB`
+        * Can be used for creation color blind effect, base for night vision effect etc.
+        * Request client with protocol version 49 or higger.
```

Comment:
Should the documentation mention that the matrix is applied before the saturation and tone mapping?

And should it mention the behaviour with out-of-bounds colours and if this behaviour is subject to change?